python/unblob/handlers/executable/elf.py

```diff
@@ -62,6 +139,12 @@ def extract(self, inpath: Path, outdir: Path):
             # other handlers might burn some cycles on the file as well.
 
 
+def extract_upx(inpath: Path, outdir: Path):
+    extractor = Command("upx", "-d", "{inpath}", "-o{outdir}/upx-decompressed.elf")
```

Comment:
I think this would be nicer:

```suggestion
    extractor = Command("upx", "-d", "{inpath}", "-o{outdir}/{inpath.stem}.elf")
```